supportedBackends/local/src/test/scala/cromwell/backend/impl/local/LocalJobExecutionActorSpec.scala

```diff
@@ -19,11 +19,10 @@ import wdl4s.values._
 
 import scala.concurrent.duration._
 
-class LocalJobExecutionActorSpec extends FlatSpec with BackendTestkitSpec with MockitoSugar with TestFileUtil with TableDrivenPropertyChecks {
+class LocalJobExecutionActorSpec extends FlatSpec with BackendTestkitSpec with MockitoSugar with TestFileUtil
```

Comment:
I thought for sure "MockitoSugar" can't be a real thing.....its nice to be proven wrong once in a while 😛